core/chains/evm/client/errors.go

```diff
@@ -118,15 +127,13 @@ var nethermind = ClientErrors{
 	NonceTooLow: regexp.MustCompile(`(: |^)OldNonce$`),
 
 	// FeeTooLow/FeeTooLowToCompete: Fee paid by this transaction is not enough to be accepted in the mempool.
```

Comment:
I think we change the comment here as well.

Reply:
Shouldn't it be updated to document `TerminallyUnderpriced` now?

Reply:
Wrong again - the name collisions between the fields and the errors strings is what is tripping me up.
Would it be more clear to re-write these are traditional godoc comments that begin the field name instead? nbd I guess :shrug: